include/hw_isolation_record/manager.hpp

```diff
@@ -231,6 +251,22 @@ class Manager :
      *       for all isolated hardware that is stored in the preserved location.
      */
     void createEntryForRecord(const openpower_guard::GuardRecord& record);
+
+    /**
+     * @brief Update the given dbus entry object for isolated hardware record
```

Comment:
may be rephrase this and other updateEntry alsmost looks like the same, if possible

Reply:
I have added the `@note` section to describe the roles of this function, let me know if that's not making sense.